package.json

```
@@ -0,0 +1,19 @@
+{
+  "name": "dev-portal",
+  "version": "1.0.0",
+  "description": "This is the content that hosts the site https://tutorials.aaf.edu.au. The site is built with [Jekyll](https://jekyllrb.com/) and hosted on [GitHub Pages](https://pages.github.com/).",
+  "main": "index.js",
+  "directories": {
+    "doc": "docs"
+  },
+  "scripts": {
+    "start-detached": "cd docs && bundle exec jekyll serve --detach",
+    "pa11y-ci:sitemap": "pa11y-ci --sitemap http://localhost:4000/sitemap.xml --sitemap-exclude \"/*.pdf\" --chrome-flags=\"--no-sandbox\""
```

CI is throwing: `error: unknown option '--chrome-flags=--no-sandbox'`

```suggestion
    "pa11y-ci:sitemap": "pa11y-ci --sitemap http://localhost:4000/sitemap.xml --sitemap-exclude \"/*.pdf\""
```

---

Done, I've removed it.